src/UCTNode.cpp

```diff
@@ -208,16 +208,17 @@ void UCTNode::dirichlet_noise(float epsilon, float alpha) {
 }
 
 void UCTNode::randomize_first_proportionally() {
-    auto accum = std::uint32_t{0};
+    auto accum = 0;
```

Comment:
This really wants to be a 64-bit but it requires a randuint64(arg), see the remarks to the Zobrist pull.

I guess our current tree limit means we can't hit this being > 32-bit signed int anyway.

Reply:
we use int which would be like 2 billion nodes or ~10 million playouts which I think is more than anyone is doing right now :)

I'll work on randuint64(arg) tomorrow